src/ProjectOrigin.WalletSystem.IntegrationTests/TestClassFixtures/JwtTokenIssuerFixture.cs

```diff
@@ -3,27 +3,30 @@
 using System.IO;
 using System.Security.Claims;
 using System.Security.Cryptography;
+using System.Text.Json;
 using AutoFixture;
 using Grpc.Core;
 using Microsoft.IdentityModel.Tokens;
 
 namespace ProjectOrigin.WalletSystem.IntegrationTests.TestClassFixtures;
 
-public class JwtTokenIssuerFixture : IDisposable
+public sealed class JwtTokenIssuerFixture : IDisposable
 {
-    public string Issuer { get; init; } = "TestIssuer";
-    public string Audience { get; } = "WalletSystem";
-    public int ExpirationMinutes { get; } = 60;
-    public string PemFilepath { get; }
+    private readonly RSA _rsa = RSA.Create();
+
+    public string KeyType => "RSA";
+    public string Algorithm => SecurityAlgorithms.RsaSha256;
+    public byte[] PublicKeyInfo => _rsa.ExportSubjectPublicKeyInfo();
+    public int ExpirationMinutes => 60;
```

Comment:
Is this a standard timeframe?

Reply:
for test jwt´s, I think an hour is sufficient.